README.md

```diff
@@ -57,7 +57,7 @@ This section will layout a few examples gradually going from very simple to more
 Firstly install the `locutus`:
 
 ```
-go get github.com:brancz/locutus
+go get github.com:rollandf/locutus
```

Comment:
can you revert this back?

Reply:
Yes, sure. I messed up with the PR.
I will send a fix soon.